tests/NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests/Features/Order Update Commencement Date.feature

```diff
@@ -0,0 +1,74 @@
+﻿Feature: Order Update Commencement Date
+    As a Buyer
+    I want to update an order's commencement date
+    So that I can keep an order up to date
+
+Background:
+    Given Orders exist
+        | OrderId    | Description         | OrderStatusId | Created    | LastUpdated | LastUpdatedByName | LastUpdatedBy                        | OrganisationId                       |
+        | C000014-01 | Some Description    | 1             | 11/05/2020 | 11/05/2020  | Bob Smith         | 335392e4-4bb1-413b-9de5-36a85c9c0422 | 4af62b99-638c-4247-875e-965239cd0c48 |
+        | C000014-02 | Another Description | 2             | 05/05/2020 | 09/05/2020  | Alice Smith       | a11a46f9-ce6f-448a-95c2-fde6e61c804a | 4af62b99-638c-4247-875e-965239cd0c48 |
+    And the user is logged in with the Buyer role for organisation 4af62b99-638c-4247-875e-965239cd0c48
+
+@5322
+Scenario: 1. Updating an orders commencement date to be today
+    Given the user sets the commencement date to today
+    When the user makes a request to update the commencement date with the ID C000014-01
+    Then a response with status code 204 is returned
+    And the order commencement date for order with id C000014-01 is set to today
```

Comment:
Worth noting Simon added the steps in for this, might be worth adding/refactoring

    And the order with orderId C000014-01 is updated in the database with data
        | LastUpdatedBy                        | LastUpdatedByName |
        | 7b195137-6a59-4854-b118-62b39a3101ef | Bob Smith         |
    And the order with orderId C000014-01 has LastUpdated time present and it is the current time